src/applications/static-pages/dependency-verification/components/dependencyVerificationFooter.jsx

```diff
@@ -1,7 +1,25 @@
 import React from 'react';
 
-const DependencyVerificationFooter = () => {
-  return <div>This is the footer where the buttons go</div>;
+const DependencyVerificationFooter = ({
+  handleClose,
+  handleCloseAndUpdateDiaries,
+}) => {
+  return (
+    <div className="medium-screen:vads-u-display--flex">
+      <button
+        className="usa-button-secondary"
+        onClick={handleCloseAndUpdateDiaries}
+      >
+        This is correct
+      </button>
+      <button onClick={handleClose}>Make changes</button>
+      <div className="vads-u-display--flex vads-u-flex--1 medium-screen:vads-u-justify-content--flex-end vads-u-align-items--center">
+        <a tabIndex="0" role="button" onClick={handleClose}>
```

Comment:
Can we make this a real `<button>`? Links without `href` values are technically reported as disabled and adding `role="button"` means we need to write extra JS to make sure this action also happened on spacebar press.

Reply:
After speaking with PM and designer, we've decided to just remove this skip link for now since it actually came up in design intent review for this particular feature.

Reply:
sounds good, thanks!